autoapi/mappers/python/objects.py

```diff
@@ -100,6 +100,22 @@ def display(self):
 
         :type: bool
         """
+        try:
+            skip = (
+                self.app.emit_firstresult(
+                    "autoapi-skip-member",
+                    self._class_content,
+                    self.name,
+                    self,
+                    self.options,
```

Comment:
Should we move this until after the following `if` statements so that we can pass their result as a `skip` option to the event?

Something like

```python
skip = False
if self.is_undoc_member and ...:
    skip = True
elif self.is_private_member and ...:
    skip = True
elif ...:
    ...

skip = self.app.emit_firstresult(..., skip, self.options)
return not skip
```

Reply:
That is good point :)